test/metabase/lib/breakout_test.cljc

```diff
@@ -464,13 +464,14 @@
   (testing "Field refs that differ from what we return should still show up as selected if they refer to the same Field (#31482)"
     (doseq [[message field-ref] {;; this ref is basically what we [[lib/breakout]] would have added but doesn't
                                  ;; contain type info, shouldn't matter tho.
-                                 "correct ref but missing :base-type/:effective-type"
+                                 #_#_"correct ref but missing :base-type/:effective-type"
                                  [:field {:lib/uuid (str (random-uuid)), :join-alias "Categories"} (meta/id :categories :name)]
```

Comment:
I'm not sure. It previously failed the Malli check because the input ref is malformed. If I disable enforcement, it succeeds. The objective of the test case is to check that matching-but-not-identical breakouts get matched correctly, so exactly how the refs differ isn't vital.

Anyway it doesn't break the Malli enforcement anymore(?) so I've brought this case back.